`Jaxec.execUnsafe()` method hides error message

Recently I got the following stack trace: 

```java
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.464 s <<< FAILURE! -- in integration.JarIT
[ERROR] integration.JarIT.runsProgramFromJar(Path) -- Time elapsed: 7.840 s <<< ERROR!
java.lang.IllegalArgumentException: Non-zero exit code #1 of 'java'
	at com.yegor256.Jaxec.execUnsafe(Jaxec.java:366)
	at com.yegor256.Jaxec.exec(Jaxec.java:329)
	at integration.JarIT.lambda$runsProgramFromJar$0(JarIT.java:75)
	at com.yegor256.farea.Farea.together(Farea.java:179)
	at integration.JarIT.runsProgramFromJar(JarIT.java:35)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   JarIT.runsProgramFromJar:35->lambda$runsProgramFromJar$0:75 » IllegalArgument Non-zero exit code #1 of 'java'
```

As you can see - it's impossible to understand anything from it. I believe `Jaxec` should print stderr in case of an error.

> [@volodya-lombrozo](https://github.com/volodya-lombrozo) it prints, but you have to configure the logger: https://github.com/yegor256/jaxec/blob/master/README.md?plain=1#L46-L48

I've set logging level to `DEBUG`, and still don't see the error:

<details>
<summary>Full log with DEBUG messages</summary>

```
G[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.eolang:eo-parser:jar:1.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.codehaus.gmaven:groovy-maven-plugin is missing. @ line 178, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ------------------< org.eolang:eo-integration-tests >-------------------
[INFO] Building eo-integration-tests 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- clean:3.4.1:clean (default-clean) @ eo-integration-tests ---
[INFO] Deleting /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests/target
[INFO] 
[INFO] --- enforcer:3.5.0:enforce (enforce-java) @ eo-integration-tests ---
[INFO] Rule 0: org.apache.maven.enforcer.rules.version.RequireJavaVersion passed
[INFO] 
[INFO] --- enforcer:3.5.0:enforce (enforce-maven) @ eo-integration-tests ---
[INFO] Rule 0: org.apache.maven.enforcer.rules.version.RequireMavenVersion passed
[INFO] 
[INFO] --- buildnumber:3.2.1:create (jcabi-build-number) @ eo-integration-tests ---
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests' && 'git' 'log' '-1' '--no-merges' '--format=format:%H %aI %aE %aN'
[INFO] Working directory: /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests
[INFO] Storing buildNumber: 41bba93 at timestamp: 2025-05-22T17:09:44
[INFO] Executing: /bin/sh -c cd '/Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests' && 'git' 'symbolic-ref' 'HEAD'
[INFO] Working directory: /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests
[INFO] Storing scmBranch: 4197-fix-simian-plugin-location
[INFO] 
[INFO] --- buildnumber:3.2.1:create-timestamp (jcabi-build-number) @ eo-integration-tests ---
[INFO] 
[INFO] --- resources:3.3.1:resources (default-resources) @ eo-integration-tests ---
[INFO] Copying 0 resource from src/main/resources to target/classes
[INFO] 
[INFO] --- compiler:3.8.1:compile (default-compile) @ eo-integration-tests ---
[INFO] No sources to compile
[INFO] 
[INFO] --- resources:3.3.1:testResources (default-testResources) @ eo-integration-tests ---
[INFO] Copying 6 resources from src/test/resources to target/test-classes
[INFO] The encoding used to copy filtered properties files have not been set. This means that the same encoding will be used to copy filtered properties files as when copying other filtered resources. This might not be what you want! Run your build with --debug to see which files might be affected. Read more at https://maven.apache.org/plugins/maven-resources-plugin/examples/filtering-properties-files.html
[INFO] 
[INFO] --- compiler:3.8.1:testCompile (default-testCompile) @ eo-integration-tests ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 5 source files to /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests/target/test-classes
[INFO] 
[INFO] --- surefire:3.5.3:test (default-test) @ eo-integration-tests ---
[INFO] Tests will run in random order. To reproduce ordering use flag -Dsurefire.runOrder.random.seed=303008197721655
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running integration.JarIT
DEBUG c.y.farea.Pom - Maven POM created at target/mktmp/JarIT/runsProgramFromJar/1st-09m47s297/pom.xml (352b)
DEBUG c.y.f.DtRequisite - Directory created at target/mktmp/JarIT/runsProgramFromJar/1st-09m47s297/src/main/eo
DEBUG c.y.f.DtRequisite - File created at target/mktmp/JarIT/runsProgramFromJar/1st-09m47s297/src/main/eo/simple.eo (38b)
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Class-Path", "Main-Class", "Manifest-Version"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["EO-Dob", "EO-Revision", "EO-Version"]
DEBUG c.j.m.Manifests - 22 attribute(s) loaded ["Ant-Version", "Automatic-Module-Name", "Bundle-ActivationPolicy", "Bundle-Category", "Bundle-Description", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-NativeCode", "Bundle-RequiredExecutionEnvironment", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Main-Class", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 20 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Version", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
DEBUG c.j.m.Manifests - 20 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Version", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
DEBUG c.j.m.Manifests - 23 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Originally-Created-By", "Provide-Capability", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 23 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Multi-Release", "Originally-Created-By", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 15 attribute(s) loaded ["Bnd-LastModified", "Build-Jdk-Spec", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 19 attribute(s) loaded ["Bnd-LastModified", "Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Import-Package", "JCabi-Build", "JCabi-Date", "JCabi-Version", "Manifest-Version", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 6 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "JCabi-Build", "JCabi-Date", "JCabi-Version", "Manifest-Version"]
DEBUG c.j.m.Manifests - 6 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "JCabi-Build", "JCabi-Date", "JCabi-Version", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 19 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 23 attribute(s) loaded ["Bnd-LastModified", "Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-Description", "Bundle-DocURL", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 19 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 22 attribute(s) loaded ["Bnd-LastModified", "Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 20 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 1 attribute(s) loaded ["Manifest-Version"]
DEBUG c.j.m.Manifests - 20 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Provide-Capability", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 2 attribute(s) loaded ["Main-Class", "Manifest-Version"]
DEBUG c.j.m.Manifests - 6 attribute(s) loaded ["Automatic-Module-Name", "Built-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version"]
DEBUG c.j.m.Manifests - 6 attribute(s) loaded ["Automatic-Module-Name", "Built-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 3 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 15 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 14 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Manifest-Version", "Multi-Release", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 4 attribute(s) loaded ["Build-Jdk", "Built-By", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 14 attribute(s) loaded ["Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Export-Package", "Implementation-Version", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability"]
DEBUG c.j.m.Manifests - 16 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SCM", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Import-Package", "Manifest-Version", "Multi-Release", "Require-Capability"]
DEBUG c.j.m.Manifests - 4 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Manifest-Version", "Multi-Release"]
DEBUG c.j.m.Manifests - 5 attribute(s) loaded ["Archiver-Version", "Build-Jdk", "Built-By", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 5 attribute(s) loaded ["Archiver-Version", "Build-Jdk", "Built-By", "Created-By", "Manifest-Version"]
DEBUG c.j.m.Manifests - 14 attribute(s) loaded ["Bnd-LastModified", "Build-Jdk", "Built-By", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Manifest-Version", "Tool"]
DEBUG c.j.m.Manifests - 30 attribute(s) loaded ["Archiver-Version", "Bnd-LastModified", "Build-Jdk", "Built-By", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Build", "Implementation-Title", "Implementation-URL", "Implementation-Vendor", "Implementation-Vendor-Id", "Implementation-Version", "Include-Resource", "Manifest-Version", "Private-Package", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
DEBUG c.j.m.Manifests - 27 attribute(s) loaded ["Ant-Version", "Automatic-Module-Name", "Bundle-ClassPath", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "DynamicImport-Package", "Eclipse-BuddyPolicy", "Eclipse-ExtensibleAPI", "Export-Package", "Extension-Name", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Main-Class", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 25 attribute(s) loaded ["Ant-Version", "Automatic-Module-Name", "Bundle-ClassPath", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "DynamicImport-Package", "Eclipse-BuddyPolicy", "Export-Package", "Extension-Name", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 20 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Provide-Capability", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 19 attribute(s) loaded ["Build-Date", "Build-Revision", "Build-Time", "Built-By", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 16 attribute(s) loaded ["Automatic-Module-Name", "Bnd-LastModified", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 11 attribute(s) loaded ["Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Can-Retransform-Classes", "Export-Package", "Import-Package", "Manifest-Version", "Premain-Class", "Require-Capability"]
DEBUG c.j.m.Manifests - 13 attribute(s) loaded ["Build-Jdk-Spec", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Created-By", "Export-Package", "Main-Class", "Manifest-Version", "Multi-Release", "Tool"]
DEBUG c.j.m.Manifests - 19 attribute(s) loaded ["Agent-Class", "Build-Jdk-Spec", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Version", "Can-Redefine-Classes", "Can-Retransform-Classes", "Can-Set-Native-Method-Prefix", "Created-By", "Export-Package", "Import-Package", "Manifest-Version", "Multi-Release", "Premain-Class", "Require-Capability", "Tool"]
DEBUG c.j.m.Manifests - 23 attribute(s) loaded ["Automatic-Module-Name", "Bnd-LastModified", "Build-Jdk-Spec", "Bundle-Description", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "Dependencies", "Export-Package", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Import-Package", "Manifest-Version", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
DEBUG c.j.m.Manifests - 9 attribute(s) loaded ["Build-Jdk-Spec", "Created-By", "Implementation-Title", "Implementation-Vendor", "Implementation-Version", "Manifest-Version", "Specification-Title", "Specification-Vendor", "Specification-Version"]
INFO  c.j.m.Manifests - 64 attributes loaded from 59 stream(s) in 40ms, 64 saved, 621 ignored: ["Agent-Class", "Ant-Version", "Archiver-Version", "Automatic-Module-Name", "Bnd-LastModified", "Build-Date", "Build-Jdk", "Build-Jdk-Spec", "Build-Revision", "Build-Time", "Built-By", "Bundle-ActivationPolicy", "Bundle-Category", "Bundle-ClassPath", "Bundle-Description", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-NativeCode", "Bundle-RequiredExecutionEnvironment", "Bundle-SCM", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Can-Redefine-Classes", "Can-Retransform-Classes", "Can-Set-Native-Method-Prefix", "Class-Path", "Created-By", "Dependencies", "DynamicImport-Package", "EO-Dob", "EO-Revision", "EO-Version", "Eclipse-BuddyPolicy", "Eclipse-ExtensibleAPI", "Export-Package", "Extension-Name", "Implementation-Build", "Implementation-Title", "Implementation-URL", "Implementation-Vendor", "Implementation-Vendor-Id", "Implementation-Version", "Import-Package", "Include-Resource", "JCabi-Build", "JCabi-Date", "JCabi-Version", "Main-Class", "Manifest-Version", "Multi-Release", "Originally-Created-By", "Premain-Class", "Private-Package", "Provide-Capability", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
DEBUG c.y.farea.Farea - Log stream redirected to target/mktmp/JarIT/runsProgramFromJar/1st-09m47s297/log.txt
DEBUG c.y.Jaxec - +mvn --update-snapshots --batch-mode --fail-fast --errors clean compile jar:jar
DEBUG c.j.l.VerboseProcess - #waitFor(): waiting for stdout of Process[pid=50571, exitValue="not exited"] in Thread[#40,VrbPrc.Monitor-945314089-out,5,main]...
DEBUG c.j.l.VerboseProcess - #waitFor(): waiting for stderr of Process[pid=50571, exitValue="not exited"] in Thread[#41,VrbPrc.Monitor-945314089-err,5,main]...
WARN  c.y.MayBeSlow - We're still running runsProgramFromJar (2s), please wait...
DEBUG c.y.farea.Farea - [INFO] Error stacktraces are turned on.
DEBUG c.y.farea.Farea - [INFO] Scanning for projects...
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] -----------------------------< test:test >------------------------------
DEBUG c.y.farea.Farea - [INFO] Building test 0.0.0
DEBUG c.y.farea.Farea - [INFO]   from pom.xml
DEBUG c.y.farea.Farea - [INFO] --------------------------------[ jar ]---------------------------------
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- clean:3.2.0:clean (default-clean) @ test ---
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- eo:1.0-SNAPSHOT:register (compile) @ test ---
DEBUG c.y.farea.Farea - [INFO] Registered 1 EO sources from src/main/eo to target/eo-foreign.json, included [**.eo], excluded []
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- eo:1.0-SNAPSHOT:compile (compile) @ test ---
DEBUG c.y.farea.Farea - [INFO] 58 attributes loaded from 74 stream(s) in 37ms, 58 saved, 1035 ignored: ["Ant-Version", "Archiver-Version", "Automatic-Module-Name", "Bnd-LastModified", "Build-Date", "Build-Jdk", "Build-Jdk-Spec", "Build-Revision", "Build-Time", "Built-By", "Built-Date", "Bundle-ClassPath", "Bundle-Description", "Bundle-Developers", "Bundle-DocURL", "Bundle-License", "Bundle-ManifestVersion", "Bundle-Name", "Bundle-SCM", "Bundle-SymbolicName", "Bundle-Vendor", "Bundle-Version", "Created-By", "DynamicImport-Package", "EO-Dob", "EO-Revision", "EO-Version", "Eclipse-BuddyPolicy", "Eclipse-ExtensibleAPI", "Export-Package", "Extension-Name", "Fragment-Host", "Implementation-Build", "Implementation-Title", "Implementation-URL", "Implementation-Url", "Implementation-Vendor", "Implementation-Vendor-Id", "Implementation-Version", "Import-Package", "Include-Resource", "JCabi-Build", "JCabi-Date", "JCabi-Version", "Lints-Version", "Main-Class", "Manifest-Version", "Multi-Release", "Originally-Created-By", "Private-Package", "Provide-Capability", "Require-Capability", "Specification-Title", "Specification-Vendor", "Specification-Version", "Tool", "X-Compile-Source-JDK", "X-Compile-Target-JDK"]
DEBUG c.y.farea.Farea - [INFO] Parsed 1 new .eo sources out of 1 to XMIRs in 2s
DEBUG c.y.farea.Farea - [INFO] No programs were probed because eo.offline flag is TRUE
DEBUG c.y.farea.Farea - [INFO] No programs were pulled because eo.offline flag is TRUE
DEBUG c.y.farea.Farea - [INFO] Assemble cycle #1 (EO:1/XMIR:0/PROBED:0 -> EO:1/XMIR:1/PROBED:0), took 2s
DEBUG c.y.farea.Farea - [INFO] No new .eo sources out of 1 parsed to XMIRs
DEBUG c.y.farea.Farea - [INFO] No programs were probed because eo.offline flag is TRUE
DEBUG c.y.farea.Farea - [INFO] No programs were pulled because eo.offline flag is TRUE
DEBUG c.y.farea.Farea - [INFO] Last assemble cycle #2 (EO:1/XMIR:1/PROBED:0), took 4ms
DEBUG c.y.farea.Farea - [INFO] 2 assemble cycle(s) produced some new object(s) in 2s: EO:1/XMIR:1/PROBED:0
DEBUG c.y.farea.Farea - [INFO] Linting is skipped because eo:skipLinting is TRUE
DEBUG c.y.farea.Farea - [INFO] Runtime dependency is ignored because eo:ignoreRuntime=TRUE
DEBUG c.y.farea.Farea - [INFO] Configured Artifact: net.java.dev.jna:jna:5.14.0:jar
DEBUG c.y.farea.Farea - [INFO] net.java.dev.jna:jna:null:5.14.0 unpacked to target/eo/4-resolve/net.java.dev.jna/jna/-/5.14.0
DEBUG c.y.farea.Farea - [INFO] Found 155 new file(s) after unpacking of net.java.dev.jna:jna:5.14.0
DEBUG c.y.farea.Farea - [INFO] New 1 dependenc(ies) unpacked to target/eo/4-resolve: net.java.dev.jna:jna:5.14.0
WARN  c.y.MayBeSlow - We're still running runsProgramFromJar (6s), please wait...
DEBUG c.y.farea.Farea - [INFO] Placed 155 binary file(s) found in 1 dependencies, into target/eo/4-resolve
DEBUG c.y.farea.Farea - [INFO] Compilation process took 3s
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- eo:1.0-SNAPSHOT:transpile (compile) @ test ---
DEBUG c.y.farea.Farea - [INFO] Transpiled 1 XMIRs, created 1 Java files in target/generated-sources
DEBUG c.y.farea.Farea - [INFO] The directory added to Maven 'compile-source-root': target/generated-sources
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- resources:3.3.1:resources (default-resources) @ test ---
DEBUG c.y.farea.Farea - [INFO] skip non existing resourceDirectory /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests/target/mktmp/JarIT/runsProgramFromJar/1st-09m47s297/src/main/resources
DEBUG c.y.farea.Farea - [INFO] 
DEBUG c.y.farea.Farea - [INFO] --- compiler:3.13.0:compile (default-compile) @ test ---
DEBUG c.y.farea.Farea - [INFO] Recompiling the module because of changed source code.
DEBUG c.y.farea.Farea - [INFO] Compiling 1 source file with javac [debug target 11] to target/classes
DEBUG c.j.l.VerboseProcess - #waitFor(): process finished: Process[pid=50571, exitValue=0]
DEBUG c.j.l.VerboseProcess - Monitor interrupted
DEBUG c.j.l.VerboseProcess - Monitor interrupted
DEBUG c.j.l.VerboseProcess - Underlying process destroyed
Classpath: test-0.0.0.jar:/Users/lombrozo/.m2/repository/org/eolang/eo-runtime/1.0-SNAPSHOT/eo-runtime-1.0-SNAPSHOT.jar
DEBUG c.y.Jaxec - +java -cp test-0.0.0.jar:/Users/lombrozo/.m2/repository/org/eolang/eo-runtime/1.0-SNAPSHOT/eo-runtime-1.0-SNAPSHOT.jar -Dfile.encoding=UTF-8 -Xss64M -Xms64M org.eolang.Main simple
DEBUG c.j.l.VerboseProcess - #waitFor(): waiting for stdout of Process[pid=50612, exitValue="not exited"] in Thread[#43,VrbPrc.Monitor-1794548118-out,5,main]...
DEBUG c.j.l.VerboseProcess - #waitFor(): waiting for stderr of Process[pid=50612, exitValue="not exited"] in Thread[#44,VrbPrc.Monitor-1794548118-err,5,main]...
DEBUG c.j.l.VerboseProcess - >> Couldn't find object 'Φ.org.eolang.io' because there's no class or package 'EOorg.EOeolang.EOio'
DEBUG c.j.l.VerboseProcess - >> EOorg.EOeolang.EOio
DEBUG c.j.l.VerboseProcess - #waitFor(): process finished: Process[pid=50612, exitValue=1]
ERROR c.y.Jaxec - 
DEBUG c.j.l.VerboseProcess - Monitor interrupted
DEBUG c.j.l.VerboseProcess - Monitor interrupted
DEBUG c.j.l.VerboseProcess - Underlying process destroyed
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.580 s <<< FAILURE! -- in integration.JarIT
[ERROR] integration.JarIT.runsProgramFromJar(Path) -- Time elapsed: 7.937 s <<< ERROR!
java.lang.IllegalArgumentException: Non-zero exit code #1 of 'java'
	at com.yegor256.Jaxec.execUnsafe(Jaxec.java:366)
	at com.yegor256.Jaxec.exec(Jaxec.java:329)
	at integration.JarIT.lambda$runsProgramFromJar$0(JarIT.java:84)
	at com.yegor256.farea.Farea.together(Farea.java:179)
	at integration.JarIT.runsProgramFromJar(JarIT.java:37)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   JarIT.runsProgramFromJar:37->lambda$runsProgramFromJar$0:84 » IllegalArgument Non-zero exit code #1 of 'java'
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11.785 s
[INFO] Finished at: 2025-05-22T17:09:55+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.5.3:test (default-test) on project eo-integration-tests: 
[ERROR] 
[ERROR] See /Users/lombrozo/Workspace/EOlang/eo/eo-integration-tests/target/surefire-reports for the individual test results.
[ERROR] See dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```
</details>

Here is `logback` configuration:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!--
 * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
 * SPDX-License-Identifier: MIT
-->
<configuration>
  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
    <encoder>
      <pattern>%highlight(%-5level) %logger{15} - %msg%n</pattern>
    </encoder>
  </appender>
  <root level="DEBUG">
    <appender-ref ref="STDOUT"/>
  </root>
</configuration>
```

--- a/.gitignore
+++ b/.gitignore
@@ -10,3 +10,4 @@
 node_modules/
 target/
 .claude/
+.aidy/

--- a/src/main/java/com/yegor256/Jaxec.java
+++ b/src/main/java/com/yegor256/Jaxec.java
@@ -119,10 +119,7 @@ public final class Jaxec {
      */
     public Jaxec(final Collection<String> args, final File dir,
         final boolean chk, final InputStream input) {
-        this(
-            new ProcessBuilder().directory(dir).redirectErrorStream(true),
-            args, chk, input
-        );
+        this(new ProcessBuilder().directory(dir), args, chk, input);
     }
 
     /**

--- a/src/test/java/com/yegor256/JaxecTest.java
+++ b/src/test/java/com/yegor256/JaxecTest.java
@@ -159,7 +159,7 @@ final class JaxecTest {
             new Jaxec("cat", "/file-is-definitely-absent")
                 .withCheck(false)
                 .exec()
-                .stdout(),
+                .stderr(),
             Matchers.containsString("No such file or directory")
         );
     }

